2023-03-14 10:51:52,670 ERROR name 'sin' is not defined
Traceback (most recent call last):
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\game.py", line 9, in <module>
    game_loop.looping()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\game_loop.py", line 69, in looping
    self._01_level.update(dt)
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\_01_main.py", line 250, in update
    self.player.update(dt)
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\player.py", line 369, in update
    self.blink()
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\player.py", line 153, in blink
    if self.wave_value():
       ^^^^^^^^^^^^^^^^^
  File "C:\Users\kevin\PycharmProjects\Super_Mario_Bros_The_Movie_The_Game\code\_01_level\player.py", line 160, in wave_value
    value = sin(pg.time.get_ticks())
            ^^^
NameError: name 'sin' is not defined
